static/js/directives/youtubevideo.js

```diff
@@ -41,20 +102,63 @@ define(['jquery', 'underscore', 'text!partials/youtubevideo.html', 'bigscreen'],
 			var player = null;
 			var playerReady = null;
 			var isPaused = null;
-			var seekDetector = null;
 			var playReceivedNow = null;
-			var prevTime = null;
-			var prevNow = null;
 			var initialState = null;
+			var sandboxFrame = $("#youtubeplayer", $element)[0];
```

Comment:
As this was the case before and ids are also used in other places in this directive, I wouldn't want to change it in this PR. Please create a ticket for this and I will change these in anew PR.